features/administration_roles/being_unable_to_edit_non_existent_administration_role.feature

```diff
@@ -0,0 +1,18 @@
+@managing_administration_roles
+Feature: Being unable to edit a non existent administration role
+   In order to have access to consistent store data
+   As an Administrator
+   I want not to be able to edit a non existent administration role
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And there is already an Administration role "Configurator" in the system
+        And this administration role has write permissions for "Configuration" and "RBAC"
+        And there is an administrator "scary.terry@nightmare.com" identified by "youCanRunButYouCannotHide"
+        And this administrator has administration role "Configurator"
+        And I am logged in as "scary.terry@nightmare.com" administrator
+
+    @ui
+    Scenario: Try to edit a non existent administration role
+        When I want to edit a non existent administration role
```

Comment:
We could be more specific, something like `I want to edit a "System Governor" administration role"`

Reply:
Since we need to pass an id when trying to open the update page, it would look a bit weird if we specified `System Governor` administration role name in the scenario and then use a random, non-existent id in the step.

Reply:
Agreed 👌